src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const error = 'Bzzz... Error!';
+
+  return (a) => {
+    count++;
+
+    return (b) => {
+      if (count > 3 & count % 2 === 0) {
+        return error;
+      }
+
+      const sum = a + b;
```

Comment:
Since we are using this value only once, there is no need to create a variable for it. Simply return the calculation